templates/build/news-feed-base.build.hbs

```diff
@@ -57,6 +57,16 @@
       \{{#if filtersEnabled}}
       <i class="fa fa-sliders" ></i>
       \{{/if}}
+      \{{#if sortEnabled}}
+      <div class="sort-group btn-group">
+        <i class="fa fa-sort-amount-desc" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" aria-hidden="true"></i>
+        <ul class="dropdown-menu list-sort sort-list-news">
```

Comment:
Is it necessary that we use a class name that's specific to the LFD layout? This UI DOM is contained in other containers that have layout specific classes, so it should be sufficient for any kind of selector targeting.

```suggestion
        <ul class="dropdown-menu list-sort">
```

Reply:
@tonytlwu We can remove this class, all we need is change `resetSortIcons` function.
And if we remove this class that is specific for layout, may I suggest moving this [function](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/426/files/2a8ab1169c25d50f7b90edf4b467dd4010ca9c98#diff-f578a21548d18676ae0271249fa231eeR161) to the Utils.js and made it common to all layouts?

Reply:
@YaroslavOvdii I don't mind. Go ahead and make it DRY.

Reply:
This still needs action